File: log-library/src/main/java/cn/maoyanluo/log_library/LogUtils.kt
package cn.maoyanluo.log_library

import android.util.Log

class LogUtils {

    companion object {

        fun i(tag: String, msg: String) = Log.i(tag, msg)
        fun d(tag: String, msg: String) = Log.d(tag, msg)
        fun w(tag: String, msg: String) = Log.w(tag, msg)
        fun e(tag: String, msg: String) = Log.e(tag, msg)

        fun i(tag: String, msg: String, t: Throwable) = Log.i(tag, msg, t)
        fun d(tag: String, msg: String, t: Throwable) = Log.d(tag, msg, t)
        fun w(tag: String, msg: String, t: Throwable) = Log.w(tag, msg, t)
        fun e(tag: String, msg: String, t: Throwable) = Log.e(tag, msg, t)

    }


}

File: Windows/GameControllerSimulator/LogLibrary/LogUtils.cs
using System;
using System.Diagnostics;

namespace LogLibrary
{
    public static class LogUtils
    {
        public static void I(string tag, string msg, Exception ex = null)
        {
            Debug.WriteLine($"[info]-[{DateTime.UtcNow.ToString()}]-[{tag}]-[{msg}]-[{ex?.Message}]-[{ex?.StackTrace}]");
        }

        public static void D(string tag, string msg, Exception ex = null)
        {
            Debug.WriteLine($"[debug]-[{DateTime.UtcNow.ToString()}]-[{tag}]-[{msg}]-[{ex?.Message}]-[{ex?.StackTrace}]");
        }

        public static void W(string tag, string msg, Exception ex = null)
        {
            Debug.WriteLine($"[warning]-[{DateTime.UtcNow.ToString()}]-[{tag}]-[{msg}]-[{ex?.Message}]-[{ex?.StackTrace}]");
        }

        public static void E(string tag, string msg, Exception ex = null)
        {
            Debug.WriteLine($"[error]-[{DateTime.UtcNow.ToString()}]-[{tag}]-[{msg}]-[{ex?.Message}]-[{ex?.StackTrace}]");
        }


    }
}
